chore: Issue template + readme updates. (#771)
launchdarkly/js-core: packages/telemetry/browser-telemetry/README.md

Author: The readme content will need updated when the 4.x SDK is released.

Also the current content is minimal, but should be enough for getting people going with metrics.

## Compatibility

This package is compatible with the `launchdarkly-js-client-sdk` version 3.4.0 and later.

## Setup

### For error metric collection only

```
import { initialize } from "launchdarkly-js-client-sdk";
import { initTelemetry, register } from "@launchdarkly/browser-telemetry";

// Initialize the telemetry as early as possible in your application.

> // Errors can be missed if they occur before the telemetry is initialized.

Reviewer: ```suggestion
// Errors will be missed if they occur before the telemetry is initialized.
```
they'll always be missed if they happen before initialization, yes?

> // For metrics only breadcrumbs and stack traces are not required.

Reviewer: ```suggestion
// If you are collecting error metrics only, breadcrumbs and stack traces are not required.
```
or if you prefer the existing terser tone, really only the `,` is required for parsing

```suggestion
// For metrics only, breadcrumbs and stack traces are not required.
```

initTelemetry({breadcrumbs: false, stack: false});

// Initialize the LaunchDarkly client.
const client = initialize('sdk-key', context);

// Register the client with the telemetry instance.
register(client);
```

### For error monitoring + metric collection

```
import { initialize } from "launchdarkly-js-client-sdk";
import { initTelemetry, register, inspectors } from "@launchdarkly/browser-telemetry";

// Initialize the telemetry as early as possible in your application.

> // Errors can be missed if they occur before the telemetry is initialized.

Reviewer: ```suggestion
// Errors will be missed if they occur before the telemetry is initialized.
```
should match line 33
